src/main.rs

```diff
@@ -221,10 +221,11 @@ async fn main() {
     let recent_ips = Arc::new(Mutex::new(VecDeque::new()));
     let client_counts_by_ip = Arc::new(Mutex::new(HashMap::new()));
 
-    let raw_listener = TcpListener::bind("0.0.0.0:12345").await.unwrap();
+    // TODO: Add an option to listen on 0.0.0.0, for local-playing.md
```

Comment:
```suggestion
    // FIXME: Add an option to listen on 0.0.0.0, for local-playing.md
    // Can't be always 0.0.0.0, because in production we avoid unnecessary non-localhost listening
```